lib/src/persist/adapter.dart

```diff
@@ -1,40 +1,41 @@
-import 'package:casbin/src/model/model.dart';
+// Copyright 2018-2021 The Casbin Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import '../model/model.dart';
```

Comment:
Try to be consistent with the style used in this file and others, if you think package imports are too verbose feel free to change them into relative imports but do so in a separate PR and refactor all of the codebase so it won't be inconsistent.

Reply:
See also: https://dart-lang.github.io/linter/lints/prefer_relative_imports.html
I'm going to change them in following prs XD